nevergrad/functions/control/mujoco.py

```diff
@@ -20,26 +20,49 @@ class GenericMujocoEnv:
         Standard deviation of state values of multiple independent runs.
     num_rollouts: int
         number of independent runs.
+    activation: str:
+        activation function
+    layer_rescaling_coef: float
+        Scaling coefficient of output layers
     random_state: int or None
         random state for reproducibility in Gym environment.
     """
 
-    def __init__(self, env_name, state_mean, state_std, num_rollouts, random_state):
-        self.mean = np.array(state_mean)
-        self.std = np.array(state_std)
+    def __init__(self, env_name, state_mean, state_std, num_rollouts,
+                 activation, layer_rescaling_coef, noise_level, random_state):
+        self.mean = state_mean
+        self.std = state_std
         self.env = gym.make(env_name)
         self.num_rollouts = num_rollouts
-        self.env.seed(random_state)
```

Comment:
I was ready to merge, however I just noticed that the `random_state` has a very different meaning than before, @herilalaina is that expected?

Reply:
Yes, this is expected. Seeding of gym environment is now handled in `_simulate` (according to `deterministic_sim`). But I still have to pass random generator to `GenericMujocoEnv` for noise generation.